### DISCUSSION TITLE
Name change proposal discussion

### DISCUSSION THREAD

**Discussion by eous** [ggml-org/llama.cpp#437] [General]
https://twitter.com/theshawwn/status/1638925249709240322

**Comment by nazthelizard122** [comment]
invite?

**Comment by Green-Sky** [comment]
@eous please provide info, a discord link can only be used as a reference, and only by those who actually use discord.

**Comment by eous** [comment]
Apologies, didn't notice discord hijacked the link to twitter. Its been updated.

**Comment by Green-Sky** [comment]
I *think* we are fine. As long as we don't post any links/torrents to the weights.
I also speculate, that lora adapters (most right now are based on alpaca data) are fine.
